CROSSSECTOR/Consumption.Point/dataModel.ConsumptionCost/ADOPTERS.yaml

```diff
@@ -1,10 +1,10 @@
-description: This is a compilation list of the current adopters of the data model [Data model] of the Subject [Subject].  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
-currentAdopters:
--
- adopter: 
- description: 
- mail: 
- organization: 
- project: 
- comments: 
+description: This is a compilation list of the current adopters of the data model [Data model] of the Subject [Subject].  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```

Comment:
please fill the fields and remove the templated content

Reply:
Added new data for ADOPTERS.yaml